platform/rct/Libraries/CMSIS/Device/ST/STM32F30x/Source/Templates/system_stm32f30x.c

```diff
@@ -201,11 +201,6 @@ void SystemInit(void)
        AHB/APBx prescalers and Flash settings ----------------------------------*/
     SetSysClock();
 
-#ifdef VECT_TAB_SRAM
-    SCB->VTOR = SRAM_BASE | VECT_TAB_OFFSET; /* Vector Table Relocation in Internal SRAM. */
-#else
-    SCB->VTOR = FLASH_BASE | VECT_TAB_OFFSET; /* Vector Table Relocation in Internal FLASH. */
```

Comment:
???  I'm assuming its supposed to be out of flash now since that is where the bootloader is?  Or am I wrong?  Just want to understand this better.